src/main/resources/application.yml

```diff
@@ -26,6 +26,13 @@ aws:
         secret-key:
       attributes:
         max-messages: 10
+    case-migrator:
```

Comment:
As you're using a `migration` label to dictate the queue secrets at deployment time, this queue isn't required as it will use the mainline queue.

Reply:
Removed.

Reply:
This isn't needed at all as case creator will effectively take the case-creator queue as the listener. You dictate in the deployment that its the case-migrator queue.